### PR TITLE
Issue #95 Handle trailing slash in Urls with 'Equals' Operator

### PR DESCRIPTION
**Pull request by harrysingh**
@blunderboy 
This handles only equals case. I have tested the changes and they work as expected.

### CODE REVIEW

File: src/background/background.js [hunk 1]
```diff
@@ -125,6 +125,9 @@ BG.Methods.modifyHeaders = function(originalHeaders, headersTarget, details) {
  */
 BG.Methods.matchUrlWithRuleSource = function(sourceObject, destination, url) {
   var operator = sourceObject.operator,
+    SLASH = '/',
```
Comment (blunderboy) [inline src/background/background.js]: We may need SLASH constant at other places too. How about moving it inside shared.js where other constants are also placed.
    RQ.STRING_CONSTANS.SLASH

File: src/background/background.js [hunk 2]
```diff
@@ -134,9 +137,18 @@ BG.Methods.matchUrlWithRuleSource = function(sourceObject, destination, url) {
       return null;
     }
   }  
-    
+
+  if (url.lastIndexOf(SLASH) === url.length - 1) {
```
Comment (blunderboy) [inline src/background/background.js]: lastIndexOf is O(n) operation and we can just use

    if (url.charAt(url.length-1) === RQ.STRING_CONSTANTS.SLASH) {
      urlWithoutTrailingSlash = url;
    }

**Comment by blunderboy** [comment]
Please submit unit test cases for this issue.

File: src/background/background.js (hunk 1 above)
Comment (harrysingh) [inline src/background/background.js]: Done

File: src/background/background.js (hunk 2 above)
Comment (harrysingh) [inline src/background/background.js]: done

**Comment by harrysingh** [comment]
@blunderboy updated the PR after fixing review comments and adding a unit test case.

**Comment by blunderboy** [comment]
@harrysingh Looks good. Merging this PR.

**Pull request merged**